cdk/package.json

```diff
@@ -1,29 +1,76 @@
 {
   "name": "cdk",
-  "version": "0.1.0",
+  "version": "0.0.0",
+  "private": true,
   "scripts": {
-    "build": "tsc --noEmit",
-    "test": "jest --runInBand --detectOpenHandles",
+    "build": "tsc",
+    "test": "jest",
+    "test-update": "jest -u",
     "format": "prettier --write \"{lib,bin}/**/*.ts\"",
     "lint": "eslint lib/** bin/** --ext .ts --no-error-on-unmatched-pattern",
-    "generate": "cdk synth --path-metadata false --version-reporting false",
-    "diff:code": "cdk diff --path-metadata false --version-reporting false Prism-CODE",
-    "diff:prod": "cdk diff --path-metadata false --version-reporting false Prism-PROD"
+    "synth": "cdk synth --path-metadata false --version-reporting false",
+    "diff": "cdk diff --path-metadata false --version-reporting false"
   },
   "devDependencies": {
-    "@guardian/cdk": "57.0.0",
+    "@guardian/cdk": "59.5.0",
     "@guardian/eslint-config-typescript": "^10.0.1",
+    "@guardian/prettier": "5.0.0",
+    "@guardian/tsconfig": "^1.0.0",
     "@types/jest": "^29.5.12",
-    "@types/node": "20.12.7",
-    "aws-cdk": "2.136.1",
-    "aws-cdk-lib": "2.136.1",
+    "@types/node": "22.5.4",
+    "aws-cdk": "2.157.0",
+    "aws-cdk-lib": "2.157.0",
     "constructs": "10.3.0",
     "eslint": "^8.57.0",
     "jest": "^29.7.0",
-    "prettier": "^3.2.5",
-    "source-map-support": "^0.5.21",
-    "ts-jest": "^29.1.2",
+    "prettier": "^3.3.3",
+    "source-map-support": "^0.5.20",
+    "ts-jest": "^29.2.5",
     "ts-node": "^10.9.2",
     "typescript": "5.3.3"
+  },
+  "prettier": "@guardian/prettier",
+  "jest": {
+    "testMatch": [
+      "<rootDir>/lib/**/*.test.ts"
+    ],
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest"
+    },
+    "setupFilesAfterEnv": [
+      "./jest.setup.js"
+    ]
+  },
+  "eslintConfig": {
+    "root": true,
+    "env": {
+      "node": true,
+      "jest": true
+    },
+    "extends": [
+      "@guardian/eslint-config-typescript"
+    ],
+    "parserOptions": {
+      "ecmaVersion": 2020,
+      "sourceType": "module"
+    },
+    "plugins": [
+      "@typescript-eslint"
+    ],
+    "rules": {
+      "@typescript-eslint/no-inferrable-types": 0,
+      "import/no-namespace": 2,
+      "quotes": [
+        "error",
+        "double"
+      ]
```

Comment:
It looks like the departmental preference is to use single quotes. Overriding that here to keep the diff in this PR as minimal as possible, and thus easier to review. This change will be reverted as it's own PR.

Reply:
https://github.com/guardian/prism/pull/946